vignettes/pinp.Rmd

```diff
@@ -287,7 +287,7 @@ be used around a wide table structure.
 
 ## widetext
 
-The `\begin{widetext*} ... \end{widetext*}` environment can be used to
+The `\begin{widetext} ... \end{widetext}` environment can be used to
```

Comment:
I do not think this is correct.  We carried this over from PNAS and I believe it to be in the cls file.